### DISCUSSION TITLE
Combine data from two domain

### DISCUSSION THREAD

**Discussion by mehulkar** [plausible/analytics#2346] [Anything Else]
I have a site deployed on multiple domains and would like to see the data together. I see that I can send the data to two domains by adding a comma separated `data-domain`: https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time, but that means the data shows up in two different dashboards. I want to see it all together, ideally.

**Comment by metmarkosaric** [comment]
thanks for the feedback @mehulkar! you could use the same snippet on different sites. see https://plausible.io/docs/plausible-script#can-i-use-the-same-snippet-across-my-domain-and-subdomain

**Comment by mehulkar** [comment]
Hmm, do I still specify `data-domain=a.com,b.com` in that case? I was not seeing data from `b.com` without that

**Reply by metmarkosaric** [reply]
you don't need to send data to multiple dashboards if you don't want to have individual dashboards for individual sites too. if you use the same snippet on multiple sites then all data will be on one dashboard but it will be combined so you won't be able to isolate stats from one site from another site. this is why it might be useful to have something like the roll up view instead https://plausible.io/docs/plausible-script#is-there-a-roll-up-view.
